Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -3440,3 +3474,34 @@ void webkitWebViewBaseCallAfterNextPresentationUpdate(WebKitWebViewBase* webView
 {
     webkitWebViewBaseNextPresentationUpdateMonitorStart(webViewBase, WTFMove(callback));
 }
+
+#if USE(GTK4)
+void webkitWebViewBaseSetPlugID(WebKitWebViewBase* webViewBase, const String& plugID)
+{
+#if defined(GTK_ACCESSIBILITY_ATSPI)
+    WebKitWebViewBasePrivate* priv = webViewBase->priv;
+
+    if (priv->socketAccessible) {
+        gtk_accessible_set_accessible_parent(priv->socketAccessible.get(), nullptr, nullptr);
+        priv->socketAccessible = nullptr;
+    }
+
+    Vector<String> tokens = plugID.split(':');
+    ASSERT(tokens.len() == 2);
```

Comment:
I would use RELEASE_ASSERT() here to ensure we always crash if the assertion is violated.

Normal ASSERT() is good too, but unfortunately it gets compiled out in release builds.

Reply:
Done, thanks